src/main/java/seedu/address/storage/JsonAdaptedProperty.java

```diff
@@ -58,11 +53,10 @@ public JsonAdaptedProperty(@JsonProperty("name") String name, @JsonProperty("pho
      */
     public JsonAdaptedProperty(Property source) {
         name = source.getName().fullName;
-        seller = source.getSeller().fullName;
-        price = source.getPrice().value.toString();
-        phone = source.getPhone().value;
-        email = source.getEmail().value;
         address = source.getAddress().value;
+        seller = new JsonAdaptedPerson(source.getSeller().getName().fullName,
+                source.getSeller().getPhone().value, source.getSeller().getEmail().value);
```

Comment:
Why not use the other constructor? 
`new JsonAdaptedPerson(source.getSeller())`

Reply:
Yea that's better for sure.